macos/FluentUI/DynamicColor/DynamicColor.swift

```diff
@@ -28,7 +28,7 @@ public class DynamicColor: NSObject {
 	}
 
 	/// resolves color based on theme
-	@objc func resolvedColor(_ appearance: NSAppearance? = nil) -> NSColor {
+	@objc public func resolvedColor(_ appearance: NSAppearance? = nil) -> NSColor {
 		guard let appearance = appearance else {
 			return self.light
 		}
```

Comment:
Completely off topic for this review, but if you wanted to make DynamicColor.swift be an actual NSColor instead of an NSObject, you can look into what we do in react-native-macos (https://github.com/microsoft/react-native-macos/blob/master/React/Base/macOS/RCTDynamicColor.m)
It's not the prettiest though :P

Reply:
Thanks Saad, I really like the idea. And code is not that bad :)

I initially had this as an extension but then I changed it to `NSObject` to have parity with iOS. At some point I would like to consolidate code across platforms for reusability and keep the interface same helps to accomplish it down the line.